File: 2 - Condições/condicoes3.c
//Escreva um programa que leia um número e mostre se ele é positivo.

/*
* Author:  Igor Vinicius Freitas de Souza
* GitHub: https://github.com/igor1043
* E-mail: igorviniciusfreitasouza@gmail.com
*/

#include <stdio.h>

int main()
{
  int numero;
  scanf("%d", &numero);
  if (numero >= 0)
  {
    printf("o numero %d eh positivo.", numero);
  }
  else
  {
    printf("o numero %d nao eh positivo.", numero);
  }
  return 0;
}

File: 2 - Condições/condicoes3.java
//Escreva um programa que leia um número e mostre se ele é positivo.

/*
* Author:  Igor Vinicius Freitas de Souza
* GitHub: https://github.com/igor1043
* E-mail: igorviniciusfreitasouza@gmail.com
*/

import java.util.Scanner;

public class condicoes3 {

  public static void main(String[] args) {
    System.out.println("Digite um número: ");
    Scanner scan = new Scanner(System.in);
    int numero = scan.nextInt();
    if (numero >= 0) {
      System.out.println("O número é positivo");
    } else {
      System.out.println("O número não é positivo");
    }
  }
}
